src/server/scripts/EasternKingdoms/zone_silverpine_forest.cpp

```diff
@@ -1,139 +1,2148 @@
+
+                    _events.ScheduleEvent(EVENT_ACTION + 18, 1s);
                     break;
-                case 16:
-                    Talk(SAY_RANE_ANSWER);
+                }
+
+                case EVENT_ACTION + 18:
+                {
+                    if (Creature* ivar = ObjectAccessor::GetCreature(*me, _bloodfangGUID))
+                        ivar->AI()->Talk(8);
+
+                    _events.ScheduleEvent(EVENT_ACTION + 19, 2s);
+                    break;
+                }
+
+                case EVENT_ACTION + 19:
+                {
+                    if (Creature* rane = ObjectAccessor::GetCreature(*me, _raneGUID))
+                        rane->AI()->Talk(2);
+
+                    _events.ScheduleEvent(EVENT_ACTION + 20, 3s);
+                    break;
+                }
+
+                case EVENT_ACTION + 20:
+                {
+                    if (Creature* rane = ObjectAccessor::GetCreature(*me, _raneGUID))
+                    {
+                        rane->ExitVehicle(0);
+                        rane->GetAI()->DoAction(ACTION_RANE_JUMP_DEATH);
+                    }
+
+                    _events.ScheduleEvent(EVENT_ACTION + 21, 2s);
                     break;
-                case 17:
-                    Talk(SAY_MOVE_QUINN);
+                }
+
+                case EVENT_ACTION + 21:
+                {
+                    if (Creature* darius = ObjectAccessor::GetCreature(*me, _crowleyGUID))
+                    {
+                        if (Creature* ivar = ObjectAccessor::GetCreature(*me, _bloodfangGUID))
+                        {
+                            ivar->SetFacingToObject(darius);
+                            ivar->AI()->Talk(9);
+                        }
+                    }
+
+                    _events.ScheduleEvent(EVENT_ACTION + 22, 4s + 500ms);
+                    break;
+                }
+
+                case EVENT_ACTION + 22:
+                {
+                    if (Creature* ivar = ObjectAccessor::GetCreature(*me, _bloodfangGUID))
+                        ivar->AI()->Talk(10);
+
+                    _events.ScheduleEvent(EVENT_ACTION + 23, 6s);
+                    break;
+                }
+
+                case EVENT_ACTION + 23:
+                {
+                    if (Player* player = ObjectAccessor::GetPlayer(*me, _playerGUID))
+                    {
+                        if (Creature* darius = ObjectAccessor::GetCreature(*me, _crowleyGUID))
+                            darius->GetAI()->DoAction(ACTION_MOVE_TO_RANA);
+
+                        if (Creature* ivar = ObjectAccessor::GetCreature(*me, _bloodfangGUID))
+                            ivar->GetAI()->DoAction(ACTION_MOVE_TO_RANA);
+                    }
+
+                    _events.ScheduleEvent(EVENT_ACTION + 24, 10s);
                     break;
-                case 24:
-                    Talk(SAY_QUINN);
+                }
+
+                case EVENT_ACTION + 24:
```

Comment:
dont want to give them proper namings, epxlaining what they are doing, or what's the main thing?

EVENT_ACTION + 24:

for example: EVENT_KILL_CREDIT or EVENT_FINISH

Reply:
I'm gonna add different EVENT_NAME to specify a little bit more.